src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,88 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final double LOAD_FACTOR = 0.75;
+    private static final int MULTIPLIER = 2;
+    private static final int INITIAL_CAPACITY = 16;
+    private int currentCapacity = INITIAL_CAPACITY;
+    private int size;
+    private Node<K, V> [] elements;
```

Comment:
The declaration of the array 'elements' should use the diamond operator to maintain type inference and reduce verbosity. It should be 'Node<K, V>[] elements;' without the explicit generic types on the right-hand side.